csrc/host_ir/container.h

```diff
@@ -55,10 +55,22 @@ class HostIrContainer final : public Fusion {
 
   Stream* getDefaultStream();
 
+  void markAlias(TensorView* original, const TensorView* new_alias) {
+    if (alias_.count(original)) {
```

Comment:
```suggestion
    while (alias_.count(original)) {
```
Is this supposed to be iterative? Is there a particular reason to go up **only one** step but not further?